packages/lesswrong/lib/subscriptionUtil.ts

```diff
@@ -26,6 +26,22 @@ export function userIsDefaultSubscribed({user, subscriptionType, collectionName,
       return user.auto_subscribe_to_my_comments && document.userId===user._id;
     case subscriptionTypes.newTagPosts:
       return false
+    // TODO: this was previously missing due to `subscriptionType` being typed `any`, so was going to default branch
+    // Figure out if it should be something else
+    case subscriptionTypes.newShortform:
+      return false;
+    // TODO: this was previously missing due to `subscriptionType` being typed `any`, so was going to default branch
+    // Figure out if it should be something else
+    case subscriptionTypes.newDebateComments:
+      return false;
+    // TODO: this was previously missing due to `subscriptionType` being typed `any`, so was going to default branch
+    // Figure out if it should be something else
+    case subscriptionTypes.newPublishedDialogueMessages:
+      return false;
+    // TODO: this was previously missing due to `subscriptionType` being typed `any`, so was going to default branch
+    // Figure out if it should be something else
+    case subscriptionTypes.newDialogueMessages:
+      return true;
```

Comment:
Are you sure you want this to be `true` and not something that's conditional on being a dialogue participant?

Reply:
Yep, good catch, thanks